test/server/lib/db/query.fixtures.js

```diff
@@ -53,32 +52,47 @@ FROM (
 ) AS counts
 GROUP BY name, dataset
 ORDER BY name~'^/Team' desc, name`
-  },
+	},
 
-  usageByUser: {
-    fromDateSQL: `SELECT date_time::DATE AS date, dataset, username, count(*)::INTEGER
+	usageByUser: {
+		fromDateSQL: `SELECT date_time::DATE AS date, dataset, username, count(*)::INTEGER
```

Comment:
Check your indentation @aarongill1 - most of the changes in this file aren't necessary - you should make sure your IDE is set to the 2 space standard we use for LEV code.

Reply:
Looked OK on IDE but issue after pushing...think I have solved